parking_permits/models/order.py

```diff
@@ -369,10 +369,11 @@ def _cancel_talpa_order(self):
         headers = {
             "api-key": settings.TALPA_API_KEY,
             "namespace": settings.NAMESPACE,
+            "user": str(self.customer.user.uuid),
             "Content-Type": "application/json",
         }
         response = requests.post(
-            f"{settings.TALPA_ORDER_EXPERIENCE_API}/{self.talpa_order_id}/cancel",
+            f"{settings.TALPA_ORDER_EXPERIENCE_API}{self.talpa_order_id}/cancel",
```

Comment:
I would maybe use `urllib.parse.urljoin` here, as it should generate the correct URL and add forward slashes as necessary e.g.

```python
urljoin(settings.TALPA_ORDER_EXPERIENCE_API, f"/self.talpa_order_id}/cancel"
```